gamedata/alldefs_post.lua

```diff
@@ -237,6 +237,45 @@ function UnitDef_Post(name, uDef)
 			end
 		end
 
+		if modOptions.unbacom == "enabled" then
+			local function setTrue(options)
+				local set = {}
+				for _, option in ipairs(options) do
+					set[option] = true
+				end
+				return set
+			end
+			
+			local unbacomlist = setTrue({"armcom", "armcomlvl2", "armcomlvl3", "armcomlvl4", "armcomlvl5", "armcomlvl6", "armcomlvl7", "armcomlvl8", "armcomlvl9", "armcomlvl10",
+			"corcom", "corcomlvl2", "corcomlvl3", "corcomlvl4", "corcomlvl5", "corcomlvl6", "corcomlvl7", "corcomlvl8", "corcomlvl9", "corcomlvl10",
+			"legcom", "legcomlvl2", "legcomlvl3", "legcomlvl4", "legcomlvl5", "legcomlvl6", "legcomlvl7", "legcomlvl8", "legcomlvl9", "legcomlvl10",})
+			
+			if unbacomlist[name] then
+				Spring.Echo(name)
+				uDef.customparams.evolution_timer = math.floor(modOptions.unbacomleveuprate*60)
+				Spring.Echo(name)
+			end
```

Comment:
This is not how this should be done. This function already runs individually for each unit, instead you should put a customdef on these units like `isUnbacom` or something, then if that's true set the evolution timer etc. 
The logs should also be removed

Reply:
done and pushed